### DISCUSSION TITLE
CTRL+6 stopped working in Version 23.2.2.202310081633

### DISCUSSION THREAD

**Discussion by Rexw099** [dbeaver/dbeaver#21544] [Q&A] [answered]
### Description

Keyboard shortcut CTRL+6 to hide/show panel stopped working with the upgrade to 23.2.2.202310081633

### DBeaver Version

23.2.2.202310081633

### Operating System

windows

### Database and driver

All

### Steps to reproduce

Trying CTRL+6 after running a query and getting results.

**Comment by E1izabeth** [comment, marked as answer]
Please try using `Ctrl+T`. We changed this shortcut recently because of the issues with AZERTY keyboards.
Also you can change shortcuts in preferences
![image](https://github.com/dbeaver/dbeaver/assets/28875055/2228273b-357c-471b-b92a-a5ec0447b357)

**Comment by marlon-sousa** [comment]
Hello,

I have found the response, but I am an experienced programmer who knows how to search and skin through github issues and comments.
ctrl + 6 was a very important command for blind people like me because i
t is the way we can use to go back from the result grid to the script we are editing, and miost part of us will cycle between edit and result set as our working flow.

I have nothing against changing the key to ctrl + t, I will use it happily. But is this in any breaking changes document so that it could be for example indexed by google or be easily found for someone searching?

If not, I would ask you to edit the release and put this quick note in there.

**Reply by E1izabeth** [reply]
Thank you for sharing. 
Indeed, not all key shortcut changes was mentioned in the release notes https://github.com/dbeaver/dbeaver/releases/tag/23.2.1
For now I can suggest checking our wiki about key shortcuts https://github.com/dbeaver/dbeaver/wiki/Shortcuts#sql-editor
We will discuss, how we can improve the process of changing key shortcuts.
